.github/workflows/win.yml

```diff
@@ -292,6 +292,25 @@ jobs:
           cp libs/* crystal/lib/
           cp src/* crystal/src -Recurse
           rm crystal/src/llvm/ext/llvm_ext.obj
+          echo "$(pwd)\crystal" >> $env:GITHUB_PATH
```

Comment:
```suggestion
          echo "$(pwd)\\\crystal" >> ${env:GITHUB_PATH}
```

Reply:
Is there a reason an escape is needed here? [I don't see it on any of the other environment variable changes.](https://github.com/crystal-lang/crystal/blob/ba35324e57b2427b2824265e56caf9cc2936d833/.github/workflows/win.yml#L262)

I'll add the brackets though like the other environment changes.

Reply:
This is wrong. Powershell does not use backslashes for escape sequences.

Reply:
That's what I assumed he was suggesting, although you use backticks in PowerShell.

Reply:
My bad, I've forgotten we're dealing with PowerShell here.